static/js/src/advantage/subscribe/react/app.tsx

```diff
@@ -18,13 +27,13 @@ Sentry.init({
   tracesSampleRate: 1.0,
 });
 
-const stripePromise = loadStripe(window.stripePublishableKey);
+const stripePromise = loadStripe(window.stripePublishableKey ?? "");
 const oneHour = 1000 * 60 * 60;
 const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
       refetchOnWindowFocus: false,
-      refetchOnmount: false,
+      refetchOnMount: false,
```

Comment:
nice to see more TS 👍 
seems like you found a bug too? Was`refetchOnmount` with incorrect casing even working before?

Reply:
I doubt it was! Thanks TS!